### ISSUE TITLE
[OAKTrust] Home Page

### ISSUE THREAD

**Issue by devangm** [TAMULib/dspace-angular#202]
**URL:** https://oaktrust-dev.library.tamu.edu/home
**WCAG 2.1 AA compliant? :** No
**Violation:**
**Issues:** 
**Notes:**  "All of OAKTrust" reads twice with slightly different text
